src/pages/LiquidityProvider/index.tsx

```diff
@@ -172,16 +172,17 @@ const LiquidityProviders: React.FC<LiquidityProvidersProps> = ({ providers, netw
                     subTitle="CANCEL"
                     mainDisabled={isNewProviderInvalid()}
                     mainOnClick={() => {
-                      if (!newProviderEndpoint.startsWith('http')) {
-                        dispatch(addErrorToast(InvalidUrl));
-                      } else {
+                      const url = new URL(newProviderEndpoint);
+                      if (!newProviderEndpoint.includes('.onion') && url.protocol === 'https:') {
```

Comment:
Why we preventing to set .onion endpoint?

the difference is only that when onion we accept ALSO http